Content.Client/NPC/Systems/NpcFactionSpriteStateSetterSystem.cs

```diff
@@ -0,0 +1,29 @@
+
+using Content.Shared.NPC.Components;
+using Content.Shared.NPC.Events;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.NPC.Systems;
+public sealed partial class NpcFactionSpriteStateSetterSystem : EntitySystem
+{
+    [Dependency] private readonly EntityManager _entityManager = default!;
```

Comment:
Not necessary. EntitySystem has an EntityManager property, and this won't be used if the below suggestion is applied.
```suggestion
```